From f8d834d9ac1e5b26f04cebd8ca6f07ed15935e70 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sat, 29 May 2021 18:32:53 +0200
Subject: [PATCH] setup.py: fix python_requires

Fix python_requires to specify a valid version.  Apparently, combining
>= and .* is disallowed, and I'm pretty sure 'just' >=3.5 means
the same.

To reproduce the problem, you can:

    $ pip install nltk distlib
    $ python -c "import distlib.database; \
      distlib.database.DistributionPath().get_distribution('nltk')"

which yields a (not very readable) error of:

    ValueError: '.*' not allowed for '>=' constraints
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 49dc08098..7be160494 100644
--- a/setup.py
+++ b/setup.py
@@ -108,7 +108,7 @@ natural language processing.  NLTK requires Python 3.5, 3.6, 3.7, 3.8, or 3.9.""
         "Topic :: Text Processing :: Linguistic",
     ],
     package_data={"nltk": ["test/*.doctest", "VERSION"]},
-    python_requires='>=3.5.*',
+    python_requires='>=3.5',
     install_requires=[
         "click",
         "joblib",
-- 
2.31.1

